ompi/datatype/ompi_datatype_internal.h

```diff
@@ -92,9 +92,6 @@
 #define OMPI_DATATYPE_MPI_C_DOUBLE_COMPLEX        0x2B
 #define OMPI_DATATYPE_MPI_C_LONG_DOUBLE_COMPLEX   0x2C
 
-#define OMPI_DATATYPE_MPI_LB                      0x2D
```

Comment:
It is a little more complicated to remove the LB and UB markers, some functions (especially those computing the bounds and extents) would need to be carefully redesigned. Why not simply removing the MPI types and leaving the OPAL types as they are ?

Reply:
I simply removed `OMPI_DATATYPE_LB` and `OMPI_DATATYPE_UB` in `ompi/datatype` and left `OPAL_DATATYPE_LB` and `OPAL_DATATYPE_UB` unchanged in `opal/datatype`

Reply:
Then it should work.